actpool/actpool.go

```diff
@@ -412,6 +458,21 @@ func (ap *actPool) validate(ctx context.Context, selp *action.SealedEnvelope) er
 	return nil
 }
 
+func (ap *actPool) removeReplacedActs(acts []*action.SealedEnvelope) {
```

Comment:
this is same as `removeInvalidActs`, see comment below?

Reply:
as discussed, will delete the `removeReplacedActs`